app/src/androidTest/java/de/test/antennapod/ui/UITestUtils.java

```diff
@@ -119,7 +119,7 @@ public void addHostedFeedData() throws IOException {
         for (int i = 0; i < NUM_FEEDS; i++) {
             Feed feed = new Feed(0, null, "Title " + i, "http://example.com/" + i, "Description of feed " + i,
                     "http://example.com/pay/feed" + i, "author " + i, "en", Feed.TYPE_RSS2, "feed" + i, null, null,
-                    "http://example.com/feed/src/" + i, false);
+                    "https://news.google.com/news/rss" + i, false);
```

Comment:
I think there is a network call, so this has been a valid rss URL.  when the test happens [here](https://github.com/AntennaPod/AntennaPod/blob/42ad712ad17b36f5c6aa6e12bfcfe4bafe2a6ace/app/src/androidTest/java/de/test/antennapod/ui/MainActivityTest.java#L83)